praw/__init__.py

```diff
@@ -949,6 +950,19 @@ def get_popular_subreddits(self, *args, **kwargs):
         url = self.config['popular_subreddits']
         return self.get_content(url, *args, **kwargs)
 
+    def get_default_subreddits(self, *args, **kwargs):
+        """Return a get_content generator for the default subreddits.
+
+        The additional parameters are passed directly into
+        :meth:`.get_content`. Note: the `url` parameter cannot be altered.
+
+        """
+        url = self.config['default_subreddits']
+        if (len(args) < self.get_content.__code__.co_varnames.index('limit') and
+                'limit' not in kwargs):
```

Comment:
I'm not a collaborator, but, this is extremely ugly.

For the sake of consistency with all the other `get_content` based generators; this shouldn't be added. If you really think it will be a cause for confusion though, you can add the `limit=None` caveat in the docstring via something like this:

```
"""Return a get_content generator for the default subreddits.

The additional parameters are passed directly into 
:meth:`.get_content`. Note: the `url` parameter cannot be altered. 
Note: if you do not pass `limit` onto :meth:`.get_content` as `None`, 
and the reddit user preferences of your account have a listing limit 
that is less than the amount of defaults, only the amount up to 
your user preference will be retrieved.
```

But I don't think it would be confusing regardless.